### PR TITLE
Example for reading shapefiles with GDAL and displaying the geometry.

### PR DESCRIPTION
**Pull request by OgreTransporter**
I have used the simple Basic example and simply read in a shapefile. As in the layer example, I created a geometric drawing class - in this case a polygon - and used it to draw the contours from the file. To test this, I loaded the country file from Natural Earth (https://www.naturalearthdata.com/download/10m/cultural/ne_10m_admin_0_countries.zip).

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -4,6 +4,7 @@ project(QGeoView LANGUAGES C CXX)
 
 option(BUILD_EXAMPLES "Build examples" ON)
 
+find_package(GDAL CONFIG)
```
Comment (AmonRaNet) [inline CMakeLists.txt:7]: Is any way to add GDAL also to qmake project?

File: samples/shapefile/CMakeLists.txt
Comment (AmonRaNet) [inline samples/shapefile/CMakeLists.txt:1]: Maybe whole sample better to name more accurate - it is GDAL example. What about gdal-shapefile

File: CMakeLists.txt (hunk 1 above)
Comment (AmonRaNet) [reply CMakeLists.txt:7]: Maybe example from here: https://github.com/chris5287/QWidgetMap/blob/master/src/Example/Example.pro

**Review by AmonRaNet — CHANGES_REQUESTED** [review]

File: samples/shapefile/CMakeLists.txt
Comment (OgreTransporter) [reply samples/shapefile/CMakeLists.txt:1]: Done

**Review by AmonRaNet — APPROVED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (AmonRaNet) [reply CMakeLists.txt:7]: Fine - I will check then myself

**Pull request merged**